test/integration/update-account-test.js

```diff
@@ -41,12 +41,17 @@ test('sign in and change username', function (t) {
     .put('/session', function (body) {
       return body.data.attributes.password === 'newsecret'
     })
-    .reply(201, signInResponseAfterUpdate)
+    .thrice()
+    .reply(201, function () {
+      // Session is updated everytime a user puts
+      signInResponseAfterUpdate.data.id = Math.random()
```

Comment:
can we do a counter instead of `Math.random()`? Something like `var counter = 0` above Line 24, and then

```js
signInResponseAfterUpdate.data.id = 'session' + (++counter)
```

So we can do two assertions

```js
// after sign in
t.is(result.session.id, 'session1', '...')
// after update
t.is(result.session.id, 'session2', '...')
```